decred/pyproject.toml

```diff
@@ -0,0 +1,39 @@
+[tool.poetry]
+name = "decred"
+version = "0.1.0"
+description = "A Python 3 Decred toolkit."
+license = "ISC"
+homepage = "https://decred.org/"
+repository = "https://github.com/decred/tinydecred/"
+documentation = "https://github.com/decred/tinydecred/blob/master/decred"
+authors = [
+    "Brian Stafford <https://github.com/buck54321>",
```

Comment:
This must be an email, or else PyPI will reject it. Oddly enough, only the first one must be an email. Use **buck@decred.org** please. 



https://test.pypi.org/project/decred/

Reply:
Ah right, forgot to ask you about it.